libpod/networking_slirp4netns.go

```diff
@@ -314,9 +314,53 @@ func (r *Runtime) setupSlirp4netns(ctr *Container) error {
 		}
 		return r.setupRootlessPortMappingViaRLK(ctr, netnsPath)
 	}
+
+	// Calculate Slirp addresses now that we are pretty sure the command executed
```

Comment:
This block has to above `if  havePortMapping...`

Reply:
I'll fix that once I figure out how to move away from globals like you suggested above.

Reply:
Fixed: c0335e77819e23be94bb55d883e18bdd6b742195